app/_locales/ko/messages.json

```diff
@@ -1,10 +1,609 @@
 {
-  "appName": {
-    "message": "MetaMask",
-    "description": "The name of the application"
+  "accept": {
+    "message": "허용"
+  },
+  "account": {
+    "message": "계좌"
+  },
+  "accountDetails": {
+    "message": "계좌 상세보기"
+  },
+  "accountName": {
+    "message": "계좌 이름"
+  },
+  "address": {
+    "message": "주소"
+  },
+  "addToken": {
+    "message": "토큰 추가"
+  },
+  "amount": {
+    "message": "양"
+  },
+  "amountPlusGas": {
+    "message": "양 + 가스"
   },
   "appDescription": {
-    "message": "이더리움 계좌 관리",
-    "description": "The description of the application"
+    "message": "이더리움 브라우저 확장 프로그램",
+    "description": "어플리케이션 내용"
+  },
+  "appName": {
+    "message": "메타마스크",
+    "description": "어플리케이션 이름"
+  },
+  "attemptingConnect": {
+    "message": "블록체인에 접속 시도 중입니다."
+  },
+  "available": {
+    "message": "유효"
```

Comment:
I don't think this is the right word for available. It means 'valid' or 'of pertinence'